tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
Can't you run enable-rdp.ps1 again?  If not then I guess we need a single place that sets up credentials and makes them available to remotely and tests.